<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicWindow</class>
 <widget class="QWidget" name="MusicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>238</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RÁDIÓ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mp3Button">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>1</y>
     <width>238</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MP3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>202</y>
     <width>478</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VISSZA</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
